Sources/X509/Signature.swift

```diff
@@ -128,6 +128,25 @@ extension Certificate.Signature {
     }
 }
 
+@available(macOS 10.15, iOS 13, watchOS 6, tvOS 13, macCatalyst 13, visionOS 1.0, *)
+extension Certificate.Signature {
+    /// The raw byte representation of the signature.
+    @inlinable
+    public var rawRepresentation: [UInt8] {
```

Comment:
Given that the ECDSA signatures are being serialized as DER, I think this is more properly `derRepresentation`.